app/javascript/__tests__/modules/listings/__snapshots__/TableSubHeader.test.tsx.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`TableSubHeader renders TableSubHeader component 1`] = `
 <div>
-  Includes Priority Units for:
+  Includes Priority Units for %{priorities}
```

Comment:
This change may need to be reverted. The changes I made with PriorityTypes removed the `%{priorities}` from the string.